benchmarks/generate_terrain_throughput.rs

```diff
@@ -1,7 +1,6 @@
 //! Benchmarks for throughput of terrain generation.
 
 #![deny(missing_docs)]
-#![deny(warnings)]
```

Comment:
I prefer to treat warnings as errors

Reply:
It breaks the rust stability guarantee

Reply:
I have no problem with denying warnings, especially since new warnings are probably important. If someone upgrades their compiler and gets new warnings and doesn't want to fix them, they have an escape hatch by locally allowing warnings, but I don't think that this should be the default practice.